keps/sig-network/563-dual-stack/README.md

```diff
@@ -1459,7 +1497,8 @@ of this feature?**
 
 * **Will enabling / using this feature result in any new calls to the cloud 
 provider?**
-  No
+  No. Errors are surfaced when the user makes a call. We don't focus on
```

Comment:
this question isn't about metrics server, it's about calls to the cloud provider. I believe in some cases IP allocation is done by the cloud provider? In GKE we have native IPs assigned to Pods. I don't know where that allocation is done exactly, maybe @bowei knows. But it seems feasible that dual stack would require an extra call to the cloud provider to allocation one of those native IPs.